virtual_rainforest/models/animals/carcasses.py

```diff
@@ -0,0 +1,17 @@
+"""The 'carcass_module.
```

Comment:
Might write this more like "The ``carcass`` module". That will ensure that carcass ends up highlighted in the built docs

Reply:
```suggestion
"""The ``carcass`` module.
```